lib/bundler.js

```diff
@@ -374,7 +374,7 @@ if (process.versions.node) {
         opts.assets.map[key].size = fileSize;
       }
 
-      if (opts.assets.inline.includes(key)) {
+      if (['css', 'js', 'svg'].includes(key.split('.').pop())) {
```

Comment:
Since this is on server side let's use extname

Reply:
We also need to tweak this:

https://github.com/shanebo/balm/blob/acdfd4a51686af40a4b751e71c0ded1d17839581/lib/bundler.js#L378-L379

Since three file types can be inlined `cssPath` isn't a good name anymore. maybe `assetPath` instead. Also it raises the question of whether we should go off the outputs path instead and remove everything before the `/dist`